src/sentry/tasks/relay.py

```diff
@@ -54,26 +54,52 @@ def update_config_cache(
         )
 
 
-def schedule_update_config_cache(
-    generate, project_id=None, organization_id=None, public_key=None, update_reason=None
+@instrumented_task(
+    name="sentry.tasks.relay.build_config_cache",
+    queue="relay_config",
+    acks_late=True,
+    soft_time_limit=30,
+    time_limit=32,
+)
+def build_config_cache(
+    organization_id=None, project_id=None, public_key=None, update_reason=None, **kwargs
```

Comment:
this task doesn't need all these arguments.  can only be public key?

Reply:
Yes